app/code/Magento/CustomerGraphQl/Model/Resolver/Customer/CustomerDataProvider.php

```diff
@@ -13,6 +13,11 @@
 use Magento\Framework\Serialize\SerializerInterface;
```

Comment:
Looks like this class has a lot of responsibility (looks like helper in Magento 1)
Need to split the class